packages/components/src/buttons/src/Button.tsx

```diff
@@ -263,6 +263,7 @@ function Button(props: ButtonProps, ref: ForwardedRef<HTMLElement>) {
                         aria-label={stringFormatter.format("Button.spinnerAriaLabel")}
                         size={size}
                         className={styles["hop-Button__Spinner"]}
+                        {...spinnerProps}
```

Comment:
This one is a bit more complicated. You have probably seen examples in components like Select. Since we are already setting a className here, we don't want it to get overridden if the user sets a className. We should extract the className from spinnerProps and combine it with the current one.